app/code/Magento/WishlistGraphQl/Model/Resolver/AddProductsToWishlistResolver.php

```diff
@@ -0,0 +1,129 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\WishlistGraphQl\Model\Resolver;
+
+use Magento\Framework\GraphQl\Config\Element\Field;
+use Magento\Framework\GraphQl\Exception\GraphQlAuthorizationException;
+use Magento\Framework\GraphQl\Exception\GraphQlInputException;
+use Magento\Framework\GraphQl\Query\ResolverInterface;
+use Magento\Framework\GraphQl\Schema\Type\ResolveInfo;
+use Magento\Wishlist\Model\ResourceModel\Wishlist as WishlistResourceModel;
+use Magento\Wishlist\Model\Wishlist\AddProductsToWishlist;
+use Magento\Wishlist\Model\Wishlist\Config as WishlistConfig;
+use Magento\Wishlist\Model\Wishlist\Data\Error;
+use Magento\Wishlist\Model\Wishlist\Data\WishlistItemFactory;
+use Magento\Wishlist\Model\WishlistFactory;
+use Magento\WishlistGraphQl\Mapper\WishlistDataMapper;
+
+/**
+ * Adding products to wishlist resolver
+ */
+class AddProductsToWishlistResolver implements ResolverInterface
```

Comment:
As a rule, we don't prefix resolvers with the "Resolver" keyword. We have some cases but they come from the past. It's not about something critical, but with the prefix we have quite a long class/file name. 
I would not put the prefix

Reply:
Was checking the resolvers from the same folder. However, the suffixes were removed.